### PR TITLE
Fixes hidden footer in UV when tabs are present

### LINKED ISSUE #5709: Download options are cropped on UV with multiple tabs
Flipping to the "Map" tab and fires a resize and the download options re-appear.

https://maps.princeton.edu/catalog/princeton-fq977x249

Screenshot 2023-03-09 at 1 44 48 PM

### PR DESCRIPTION
**Pull request by sdellis**
Fixes the hidden footer by making the function that adds tabs asynchronous.

closes #5709

### CODE REVIEW

File: app/javascript/viewer/uv_manager.js [hunk 1]
```diff
@@ -213,8 +213,8 @@ export default class UVManager {
     this.uvElement.width(windowWidth)
     this.uvElement.height(windowHeight - titleHeight - tabHeight)
     this.uvElement.children('div').height(windowHeight - titleHeight - tabHeight)
-    if (this.uv) { this.uv.resize() }
     this.waitForElementToDisplay('button.share', 1000, this.addViewerIcons.bind(this))
+    if (this.uv) { console.log("resize"); this.uv.resize() }
```
Comment (hackartisan) [inline app/javascript/viewer/uv_manager.js:217]: Do we want to keep this console log?

**Review by hackartisan — CHANGES_REQUESTED** [review]

**Comment by hackartisan** [comment]
Adding a ref to #5709

File: app/javascript/viewer/uv_manager.js (hunk 1 above)
Comment (sdellis) [reply app/javascript/viewer/uv_manager.js]: Good catch, @hackartisan ! I removed it and pushed the commit.

**Review by hackartisan — APPROVED** [review]
I just looked at https://figgy-staging.princeton.edu/catalog/parent/2dd9208e-d3e9-4d65-b8c0-d2eae602557a/d7be88a4-9083-4d7a-89b0-0b0f92382fd9 on staging and it looks good -- assuming the footer is only supposed to be visible on the "sheets" tab -- the "map" tab is just an open seadragon panel?

@sdellis This PR should pass CI if you rebase it, and then it's ready for merge.

**Pull request merged**